tests/testsuite/version.rs

```diff
@@ -53,10 +52,18 @@ fn version_works_with_bad_target_dir() {
 fn verbose() {
     // This is mainly to check that it doesn't explode.
     cargo_process("-vV")
-        .with_stdout_contains(&format!("cargo {}", cargo::version()))
-        .with_stdout_contains("host: [..]")
-        .with_stdout_contains("libgit2: [..]")
-        .with_stdout_contains("libcurl: [..]")
-        .with_stdout_contains("os: [..]")
+        .with_stdout_data(
+            str![[r#"
+cargo [..] ([..] [..])
+release: [..]
+commit-hash: [..]
+commit-date: [..]
+host: [HOST_TARGET]
+libgit2: [..] (sys:[..] [..])
+libcurl: [..] (sys:[..] [..])
+...
+"#]]
+            .unordered(),
+        )
```

Comment:
Sorry but we're missing the assertion of `os` part. We could preserve the exact match of cargo version string, and omit ssl part with `...`, and drop `.unordered` call. What do you think?

```suggestion
        .with_stdout_data(format!(
            "\
cargo {}
release: [..]
commit-hash: [..]
commit-date: [..]
host: [HOST_TARGET]
libgit2: [..] (sys:[..] [..])
libcurl: [..] (sys:[..] [..])
...
os: [..]
",
            cargo::version()
        ))
```

Reply:
Agreed.

Reply:
apply suggestion at commit 842579e